students/John_Sekora/lesson06/mailroom4_test.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+from mailroom4 import *
+
+
+class TestMailroom4(TestCase):
+
+    def test_data(self):
+        self.assertEqual(data, {'John Smith': [400],
+                                'Bill Wilmer': [8000, 10000, 3000],
+                                'George Guy': [50],
+                                'Elizabeth Jones': [2000, 1000, 2000],
+                                'Nathan Star': [250.50, 100]})
+
+
+if __name__ == 'main':
```

Comment:
This isn't testing any of your functions.  The idea behind a unit test is that you test each function individually without the need for the wider program.  https://jeffknupp.com/blog/2013/12/09/improve-your-python-understanding-unit-testing/